services/content-watcher/libs/common/src/ipfs/ipfs.dsnp.ts

```diff
@@ -98,7 +98,7 @@ export class IPFSContentProcessor extends BaseConsumer {
       if (isBroadcast(mapRecord)) {
         announcementResponse.announcement = {
           fromId: mapRecord.fromId,
-          contentHash: bases.base16.encode(mapRecord.contentHash as never),
```

Comment:
I wonder was this done as a hack for the bug ?

Reply:
Not sure. It might have been me at some point, but I don't recall.

Reply:
I forget where this landed, spec-wise... the original DSNP spec required hex encoding for serialization of the contentHash, but I think the pending PR changes it to any multihash encoding, maybe?

Reply:
...or is that just for the DSNP content URI?

Reply:
mostly for content uri i believe, looking in social template noticed there is no hash validation done there so maybe this hack worked to mitigate bad hashes